internal/seclang/rule_parser.go

```diff
@@ -232,13 +232,24 @@ func (p *RuleParser) ParseDefaultActions(actions string) error {
 		if action.Atype == rules.ActionTypeDisruptive {
 			defaultDisruptive = action.Key
 		}
+		// SecDefaultActions can not contain metadata actions
+		if action.Atype == rules.ActionTypeMetadata {
```

Comment:
According to [Doc](https://coraza.io/docs/seclang/directives/#SecDefaultAction): `Every SecDefaultAction directive must specify a disruptive action and a processing phase and cannot contain metadata actions.`